### DISCUSSION TITLE
Autocomplete for typeahead search API. (e.g. address autocomplete)

### DISCUSSION THREAD

**Discussion by matthiasfeist** [mantinedev/mantine#5627] [Q&A] [answered]
Hi everyone!

I'm currently trying to add a address autocomplete to my project that uses Mantine as the primary UI library. I looked at the examples for the autocomplete component but I don't really understand how I would use the text that a user has entered to then send it to an API and then populate the autocomplete options with the results of that API call. And of course repeat this for each new character the user enters.

I hope someone can help me out here :) thanks so much!

**Comment by rtivital** [comment]
https://ui.mantine.dev/component/autocomplete-loading/

**Reply by matthiasfeist** [reply]
stupid question: How do I get to this example on the UI website? I'm asking because it wasn't really clear to me that the UI page has more examples 
And I wonder where on the UI page to find more examples like these :)

**Reply by rtivital** [reply]
https://ui.mantine.dev/

**Reply by matthiasfeist** [reply]
yes that I get but where on this site was the async input example?

**Reply by rtivital** [reply, marked as answer]
https://ui.mantine.dev/category/inputs/#autocomplete-loading

**Reply by matthiasfeist** [reply]
awesome! thanks!

**Comment by matthiasfeist** [comment]
awesome thanks!
